engines/sherlock/scalpel/scalpel.cpp

```diff
@@ -0,0 +1,813 @@
+				_res->addToCache("finale3.vdx", "epilogue.lib");
+				_res->addToCache("finale4.vda", "EPILOG2.lib");
+				_res->addToCache("finale4.vdx", "EPILOG2.lib");
+			}
+
+			_animation->play("finale1", 1, 0, false, 4);
+			_animation->play("finale2", 1, 0, false, 4);
+			_animation->play("finale3", 1, 0, false, 4);
+
+			_useEpilogue2 = true;
+			_animation->play("finale4", 1, 0, false, 4);
+			_useEpilogue2 = false;
+			break;
+
+		case 53:
+			// Moorehead's death / subway train
+			_res->addToCache("SUBWAY2.vda", "epilogue.lib");
+			_res->addToCache("SUBWAY2.vdx", "epilogue.lib");
+			_res->addToCache("SUBWAY3.vda", "epilogue.lib");
+			_res->addToCache("SUBWAY3.vdx", "epilogue.lib");
+
+			_animation->play("SUBWAY1", 1, 3, true, 4);
+			_animation->play("SUBWAY2", 1, 0, false, 4);
+			_animation->play("SUBWAY3", 1, 0, false, 4);
+
+			// Set fading to direct fade temporary so the transition goes quickly.
+			_scene->_tempFadeStyle = _screen->_fadeStyle ? 257 : 256;
+			_screen->_fadeStyle = false;
+			break;
+
+		case 70:
+			// Brumwell suicide
+			_animation->play("suicid", 1, 3, true, 4);
+			break;
+		default:
+			break;
+		}
+
+		// Except for the Moorehead Murder scene, fade to black first
+		if (_scene->_goToScene != 53) {
+			_events->wait(40);
+			_screen->fadeToBlack(3);
+		}
+
+		switch (_scene->_goToScene) {
+		case 52:
+			_scene->_goToScene = 27;			// Go to the Lawyer's Office
+			_map->_bigPos = Common::Point(0, 0);	// Overland scroll position
+			_map->_overPos = Common::Point(22900 - 600, 9400 + 900);	// Overland position
+			_map->_oldCharPoint = 27;
+			break;
+
+		case 53:
+			_scene->_goToScene = 17;			// Go to St. Pancras Station
+			_map->_bigPos = Common::Point(0, 0);	// Overland scroll position
+			_map->_overPos = Common::Point(32500 - 600, 3000 + 900);	// Overland position
+			_map->_oldCharPoint = 17;
+			break;
+
+		default:
+			_scene->_goToScene = 4;			// Back to Baker st.
+			_map->_bigPos = Common::Point(0, 0);	// Overland scroll position
+			_map->_overPos = Common::Point(14500 - 600, 8400 + 900);	// Overland position
+			_map->_oldCharPoint = 4;
+			break;
+		}
+
+		// Free any song from the previous scene
+		_sound->freeSong();
+		break;
+
+	case 55:
+		// Exit game
+		_screen->fadeToBlack(3);
+		quitGame();
+		return;
+
+	default:
+		break;
+	}
+
+	_events->loadCursors("rmouse.vgs");
+	_events->setCursor(ARROW);
+
+	if (_scene->_goToScene == 99) {
+		// Darts Board minigame
+		_darts->playDarts();
+		_mapResult = _scene->_goToScene = 19;	// Go back to the bar
+	}
+
+	_mapResult = _scene->_goToScene;
+}
+
+/**
+ * Takes care of clearing the mirror in scene 12, in case anything drew over it
+ */
+void ScalpelEngine::eraseMirror12() {
+	Common::Point pt((*_people)[AL]._position.x / 100, (*_people)[AL]._position.y / 100);
+
+	// If player is in range of the mirror, then restore background from the secondary back buffer
+	if (Common::Rect(70, 100, 200, 200).contains(pt)) {
```

Comment:
The only other place where I commonly check rect boundings are when checking against the _POINTS arrays. I actually experimented with converting them to use Common::Rect() form as above, but I couldn't come up with a form that didn't end up looking nice, since the converted form ended up being just as long as the original.